content/docs/sidebar.json

```diff
@@ -97,16 +88,22 @@
         ]
       },
       {
-        "label": "CI/CD for Machine Learning",
-        "slug": "ci-cd-for-machine-learning"
+        "slug": "fast-data-caching-hub",
+        "label": "Fast and Secure Data Caching Hub"
+      },
+      "data-registry",
+      {
+        "slug": "ci-cd-for-machine-learning",
+        "label": "CI/CD for Machine Learning"
       },
       {
-        "label": "Fast and Secure Data Caching Hub",
-        "slug": "fast-data-caching-hub"
+        "slug": "ml-experiment-tracking",
+        "label": "ML Experiment Tracking"
       },
-      "experiment-tracking",
-      "model-registry",
-      "data-registry"
+      {
+        "slug": "ml-model-registry",
+        "label": "ML Model Registry"
```

Comment:
Hi @rogermparent qq: where do we add words for default capitalization from sidebar slugs? I'd like to make `ml`->"ML". Thanks

Reply:
Currently the code is written in a way that only `dvc` can be uppercased. But if we want to be able uppercase more than just DVC, it should be doable with a little code refactoring! I'll open a issue for it.

Reply:
#3602 has been merged and `ml` should become `ML` automatically :)

Reply:
Awesome, thanks Julie! Should this be a feature of the docs engine in general? CML and MLEM for example could use this 🙂

Reply:
Yes! Already opened a pr for adding this feature to our doc engine theme (See gatsby-theme-iterative#22)

Reply:
`dvclive` - > "DVCLive" ? Or is that too much

Reply:
a bit too much, Jorge (how many places do we have)?

Reply:
A few, same as the other ones. But yes never mind, it complicates things conceptually.